Hunting Queries/MultipleDataSources/MultiplePasswordResetbyUser.txt

```diff
@@ -0,0 +1,61 @@
+// Name: Multiple Password Reset by user
+// Description: This query will determine multiple password resets by user across multiple data sources. 
+// Account manipulation including password reset may aid adversaries in maintaining access to credentials 
+// and certain permission levels within an environment.
+// 
+// Id: 0b9ae89d-8cad-461c-808f-0494f70ad5c4
+//
+// DataSource: #AuditLogs, #Syslog, #SecurityEvent, #OfficeActivity
+//
+// Tactics: #Discovery, #CredentialAccess
+//
+let timeframe = 1d;
+let PerUserThreshold = 5;
+let TotalThreshold = 100;
+let PasswordResetMultiDataSource =
+(union isfuzzy=truegit
+(
+//WEF Password reset events
+//4723: An attempt was made to change an account's password
+//4724: An attempt was made to reset an accounts password
+SecurityEvent
+| where TimeGenerated >= ago(timeframe)
+| where EventID in ("4723","4724")
```

Comment:
Do you want AccountType == Machine here?  It comes through in my query.

Reply:
submitted new commit with changes